dowhy/causal_refuters/dummy_outcome_refuter.py

```diff
@@ -18,59 +19,67 @@ class DummyOutcomeRefuter(CausalRefuter):
 
     Supports additional parameters that can be specified in the refute_estimate() method.
 
-    - 'dummy_outcome_type': str, None by default
-    Default is to generate random values for the treatment. If palcebo_type is "permute",
-    then the original treatement values are permuted by row.
     - 'num_simulations': int, CausalRefuter.DEFAULT_NUM_SIMULATIONS by default
     The number of simulations to be run
-    - 'random_state': int, RandomState, None by default
-    The seed value to be added if we wish to repeat the same random behavior. If we want to repeat the
-    same behavior we push the same seed in the psuedo-random generator
-    - 'outcome_function': function pd.Dataframe -> np.ndarray, None
-    A function that takes in a function that takes the input data frame as the input and outputs the outcome
-    variable. This allows us to create an output varable that only depends on the confounders and does not depend 
-    on the treatment variable.
-    Currently it supports some common functions like 
-        1. Linear Regression
-        2. K Nearest Neighbours
-        3. Support Vector Machine
-        4. Neural Network
-        5. Random Forest
-    - 'params': dict, default {}
-    The parameters that go with the outcome_function. This consists of the parameters to be passed to the sklearn objects
-    to give the desired behavior.
+    - 'pipeline': list, [noise]

```

Comment:
"transformations" is probably a better word. since all these operations transform the outcome.